frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -84,7 +84,12 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
     });
   };
 
-  useResourceValidation(values.formData.tasks, values.formData.resources, updateErrors);
+  useResourceValidation(
```

Comment:
Yeah, this hook will be going away once I finish reworking my PR... FWIW though, if it wasn't going away `useResourceValidation` taking `workspaces` is not the best maintainable use here.